### DISCUSSION TITLE
error on create EROFS IMAGES

### DISCUSSION THREAD

**Discussion by williamvyong** [LSPosed/MagiskOnWSALocal#545] [General]
after download all thing needed but error on here 

Create EROFS images
mkfs.erofs 1.0
mkfs.erofs: unrecognized option '--chunksize=4096'
usage: [options] FILE DIRECTORY

Generate erofs image from DIRECTORY to FILE, and [options] are:
 -zX[,Y]   X=compressor (Y=compression level, optional)
 -d#       set output message level to # (maximum 9)
 -x#       set xattr tolerance to # (< 0, disable xattrs; default 2)
 -EX[,...] X=extended options
 -T#       set a fixed UNIX timestamp # to all files
 --help    display this help and exit

Available compressors are: lz4hc, lz4
ERROR: Failed to make erofs image from /tmp/wsa-build-lhU6Oa5M62_/system_root_merged/vendor
Build: an error has occurred, exit

**Comment by williamvyong** [comment]
i try to upgrade erofs-utils
erofs-utils is already the newest version (1.0-3).

**Comment by Howard20181** [comment]
https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git/tree/docs/INSTALL.md

**Reply by williamvyong** [reply]
sorry im noobies can u teach me on this ?
